docker/orca-nightly/install-orca-env.sh

```diff
@@ -0,0 +1,4 @@
+wget https://raw.githubusercontent.com/intel-analytics/BigDL/main/python/requirements/orca/requirements_automl.txt && \
+conda create -y -n bigdl-orca python=3.7 && \
+source activate bigdl-orca && \
```

Comment:
Let's name the env `bigdl` in case this image will replace bigdl-spark-3.1.3 in the future?